drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -10,23 +10,46 @@ module Types
   class HmisSchema::Client < Types::BaseObject
     include Types::HmisSchema::HasEnrollments
 
+    def self.configuration
+      Hmis::Hud::Client.hmis_configuration(version: '2022')
+    end
+
     description 'HUD Client'
     field :id, ID, null: false
-    field :personal_id, String, null: false
-    field :first_name, String, null: true
-    field :last_name, String, null: true
+    hud_field :personal_id
+    hud_field :first_name
+    hud_field :last_name
     field :preferred_name, String, null: true
-    field :name_suffix, String, null: true
-    field :ssn_serial, String, null: true
-    field :dob, GraphQL::Types::ISO8601Date, 'Date of birth as format yyyy-mm-dd', null: true
+    hud_field :name_suffix
+    hud_field :name_data_quality, Types::HmisSchema::Enums::NameDataQuality
+    hud_field :dob
+    hud_field :dob_data_quality, Types::HmisSchema::Enums::DOBDataQuality
+    hud_field :ssn
+    hud_field :ssn_data_quality, Types::HmisSchema::Enums::SSNDataQuality
+    field :gender, [Types::HmisSchema::Enums::Gender], null: false
+    field :race, [Types::HmisSchema::Enums::Race], null: false
+    hud_field :ethnicity, Types::HmisSchema::Enums::Ethnicity
+    hud_field :veteran_status, Types::HmisSchema::Enums::VeteranStatus
     field :pronouns, String, null: true
     enrollments_field :enrollments, type: Types::HmisSchema::Enrollment.page_type
-    field :date_updated, GraphQL::Types::ISO8601DateTime, null: false
-    field :date_created, GraphQL::Types::ISO8601DateTime, null: false
-    field :date_deleted, GraphQL::Types::ISO8601DateTime, null: true
+    hud_field :date_updated
+    hud_field :date_created
+    hud_field :date_deleted
 
     def enrollments(**args)
       resolve_enrollments(**args)
     end
+
+    def gender
+      selected_genders = ::HUD.gender_id_to_field_name.except(8, 9, 99).select { |_id, f| object.send(f).to_i == 1 }.keys
```

Comment:
Maybe:
```suggestion
      selected_genders = ::HUD.gender_fields.except(:GenderNone).select { |f| object.send(f).to_i == 1 }
```
To better match the race setup below?

Reply:
That doesn't exactly work, because we need the numeric ids. Those are the "keys" for the GQL enum. Made this change to make it cleaner though, in the same spirit https://github.com/greenriver/hmis-warehouse/pull/2562/commits/47e22c4d06e302b5c33eb66df119e859fc34e027

Reply:
Thanks!